.prettierrc.mjs

```diff
@@ -22,18 +22,18 @@ export default {
   useTabs: false,
   embeddedLanguageFormatting: 'auto',
   endOfLine: 'auto',
-  importOrder: [
-    '^node:(.*)$',
-    '<THIRD_PARTY_MODULES>',
-    '^@ui-tars/(.*)$',
-    '^@main/(.*)$',
-    '^@shared/(.*)$',
-    '^@renderer/(.*)$',
-    '^@resources/(.*)$',
-    '^[./]',
-  ],
-  importOrderSeparation: true,
-  importOrderSortSpecifiers: true,
+  // importOrder: [
+  //   '^node:(.*)$',
+  //   '<THIRD_PARTY_MODULES>',
+  //   '^@ui-tars/(.*)$',
+  //   '^@main/(.*)$',
+  //   '^@shared/(.*)$',
+  //   '^@renderer/(.*)$',
+  //   '^@resources/(.*)$',
+  //   '^[./]',
+  // ],
+  // importOrderSeparation: true,
+  // importOrderSortSpecifiers: true,
```

Comment:
why removing this

Reply:
https://github.com/trivago/prettier-plugin-sort-imports/issues/229#issuecomment-2614411194